add sqlalchemy locations

taylor-nguyen-987 opened the pull request:
## What's New?
Please include a summary of the change and which issue is fixed. 
Please also include relevant motivation and context. 
List any dependencies that are required for this change.

I have created a Locations.py file and edited the database_wrapper.py to save the location information. 

## Fixes #55

## Type of change (pick-one)
- [ ] Bug fix (_non-breaking change which fixes an issue_)
- [X] New feature (_non-breaking change which adds functionality_)
- [ ] Breaking change (_fix or feature that would cause existing functionality to not work as expected_)
- [ ] This change requires a documentation update

## How Has This Been Tested?
Please describe the tests that you ran to verify your changes. 
Provide instructions so we can reproduce. 
Please also list any relevant details for your test configuration (hardware/operating system/etc).

## Checklist (check-all-before-merge)
_formatting help: `- [x]` means "checked' and `- [ ]` means "unchecked"_

- [ ] I documented my code according to the [Google Python Style Guide][1]

- [ ] I ran `./build_docs.sh` and the docs look fine

- [ ] I ran `./type_check.sh` and got no errors

- [ ] I ran `./format.sh` because it automatically cleans my code for me 😄 

- [ ] I ran `./lint.sh` to check for what "format" missed

- [ ] I added my tests to the `/tests` directory

- [ ] I ran `./run_tests.sh` and all the tests pass

[1]: https://google.github.io/styleguide/pyguide.html

taylor-nguyen-987 committed:
add sqlalchemy locations

mfekadu reviewed (commented):
Looks super good, thank you @taylor-nguyen-987 !

mfekadu committed:
Merge branch 'dev' into locations

mfekadu committed:
fixes longitude and latitude fields && tested manually

We ran
```
./database_wrapper.py
```

and the result was a new table (Location) created in the database with the provided data dictionary

Co-Authored-By: taylor-nguyen-987 <taylor-nguyen-987@users.noreply.github.com>
